.github/workflows/lang-npgsql.yml

```diff
@@ -42,8 +42,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ 'ubuntu-latest' ]
-        dotnet-version: [ '6.0.x', '7.0.x', '8.0.x' ]
-        npgsql-version: [ '6.0.11', '7.0.7', '8.0.3' ]
+        dotnet-version: [ '7.0.x', '8.0.x', '9.0.x' ]
```

Comment:
.NET 7 is no longer maintained while .NET 6 is (until November 2024). Would it make sense to wait for .NET 9 to be released (currently only released as RC)?

Reply:
## .NET 9

> Would it make sense to wait for .NET 9 to be released (currently only released as RC)?

We are not doing any super strong promises here. I am always happy to unlock CI slots as soon as artefacts of major releases become available, even as RC versions, so our assets can be tested on them from the very beginning.

If you don't have any objections agains that philosophy, let's embrace it?

Reply:
> Do you think we should remove it from the CI matrix?

As long as it doesn't hurt us, we can keep it.

> don't want to terminate validation of .NET 6 too early

.NET 6 is gone next month anyway, so let's remove it instead of spending time on figuring out why it failed. It is unlikely any user will complain.